### DISCUSSION TITLE
I do not get email to create new password

### DISCUSSION THREAD

**Discussion by thomasdepraetere21** [community/community#114606] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi I got various emails all ending in dokeos.com but I am unable with all of them to get a new password into Github. Can you help me?

**Comment by mairinkdev** [comment]
Have you attempted to reach out to GitHub support for assistance with resetting your password? They should be able to help you resolve the issue.
